### PR TITLE
Added a method to get pinned repositories

### PR DESCRIPTION
**Pull request by ayushanand308**
This method returns list of all the pinned repositories for a given username

### CODE REVIEW

File: src/scrape_up/github/__init__.py [hunk 1]
```diff
@@ -49,3 +49,17 @@ def get_bio(self):
         except:
             message = f"Bio not found for username {self.username}"
             return message
+        
+        
+    def get_repo(self):
+        """
```
Comment (nikhil25803) [inline src/scrape_up/github/__init__.py:55]: Hey @ayushanand308, include this method in the `endpoint.md` file

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

**Comment by ayushanand308** [comment]
I am not able to find endpoint.md in this repo? Can you help me

**Comment by nikhil25803** [comment]
> I am not able to find endpoint.md in this repo? Can you help me

Sorry it is, `documentation.md` file. ;))

**Comment by ayushanand308** [comment]
How can I remove merge conflict?

File: src/scrape_up/github/__init__.py (hunk 1 above)
Comment (ayushanand308) [reply src/scrape_up/github/__init__.py:55]: Done.

**Review by nikhil25803 — COMMENTED** [review]
Changes Approved

**Pull request merged**